packages/node_modules/@cerebral/firebase-admin/package.json

```diff
@@ -10,10 +10,12 @@
   },
   "dependencies": {
     "faye-websocket": "^0.11.1",
-    "firebase": "^3.9.0",
+    "proxyquire": "^1.7.11"
+  },
+  "peerDependencies": {
+    "firebase": "^5.4.3",
     "firebase-admin": "^4.0.4",
     "firebase-queue": "^1.6.0",
```

Comment:
@christianalfoni you forgot to remove the comma at the end of the line.
Like this Travis is erroring.